ydb/mvp/core/core_ydb.h

```diff
@@ -156,6 +157,7 @@ struct TYdbLocation {
     static TString UserToken;
     static TString CaCertificate;
     static TString SslCertificate;
+    TString MetaDatabaseTokenName;
```

Comment:
it's doesn't look like a part of every location, more like system-wide setting

Reply:
moved from location. It's static variable in TMVP now